contrib/yq/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "yq"
+pkgver = "4.40.7"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+checkdepends = ["bash", "tzdata"]
+pkgdesc = "Portable command-line YAML processor written in Go"
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT"
+url = "https://github.com/mikefarah/yq"
+source = f"{url}/archive/v{pkgver}.tar.gz"
+sha256 = "c38024d40ee37d26caba1824965d9ea1d65468f48b2bacd45647ff4f547fa59f"
+# no tests defined
+options = ["!check"]
+
+
+def post_install(self):
+    self.install_license("LICENSE")
+    for shell in ["bash", "fish", "zsh"]:
+        self.do(
+            "sh",
+            "-c",
+            f"{self.chroot_cwd}/build/yq shell-completion {shell} > yq.{shell}",
+        )
+        self.install_completion(f"yq.{shell}", shell=f"{shell}")
```

Comment:
```suggestion
        self.install_completion(f"yq.{shell}", shell=shell)
```

pretty sure you don't need to restring it